lib/smart_answer_flows/marriage-abroad.rb

```diff
@@ -733,8 +733,8 @@ def define
             end
           end
 
-          if ceremony_country == 'italy' && resident_of != 'uk'
-            phrases << :required_supporting_documents_italy
+          if resident_of != 'uk' && data_query.phrase_exists?("required_supporting_documents_#{ceremony_country}")
```

Comment:
I suppose I do mean something like that, but I'm thinking further ahead when this kind of thing would be extracted out of the flow class.

I take your point about potentially making changes affecting multiple countries harder, but I think the data should be modelled in a way that makes it easy to use & understand within the application rather than in a way that makes it easy to update. If necessary, we can then provide tools to bulk-edit multiple countries. Anyway, this is probably a discussion for another day and shouldn't affect this PR.